.github/workflows/super-linter.yml

```
@@ -0,0 +1,50 @@
+          # Full git history is needed to get a proper list of changed files within `super-linter`
+          fetch-depth: 0
+
+      - name: Lint Code Base
+        uses: github/super-linter@v4
+        env:
+          VALIDATE_ALL_CODEBASE: false
+          DEFAULT_BRANCH: "main"
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+            Terraform Apply for GorillaStack
+  runs-on:  You may pin to the exact commit or the version.
+   uses: SWO-GS/github-action-apply-on-merge@71fab58db0a839b1e9b34e87601a5e4347c52c88
```

actionlint (1.7.7)

could not parse as YAML: yaml: line 32: mapping values are not allowed in this context

(syntax-check)